src/ploigos_step_runner/step_implementers/automated_governance/rekor.py

```diff
@@ -0,0 +1,277 @@
+        with open(file_path, "rb") as f:
+            # Read and update hash string value in blocks of 4K
+            for byte_block in iter(lambda: f.read(4096), b""):
+                sha256_hash.update(byte_block)
+        return sha256_hash.hexdigest()
+
+    def create_rekor_entry( self,
+        public_key_path,
+        extra_data_file
+    ):
+        """Creates a rekor entry as a dict object.
+
+        Returns
+        -------
+        StepResult
+            Dict that contains rekor entry for upload with the cli
+        """
+        hash = self.get_file_hash(extra_data_file)
+        sig_file = extra_data_file + '.asc'
+        sig_file_path = Path(sig_file)
+        if sig_file_path.exists():
+            sig_file_path.unlink()
+        self.get_gpg_key(sig_file,extra_data_file)
+        base64_encoded_extra_data = self.base64_encode(extra_data_file)
+        rekor_entry = {
+            "kind": "rekord",
+            "apiVersion": "0.0.1",
+            "spec": {
+                "signature": {
+                    "format": "pgp",
+                    "content": self.base64_encode(sig_file),
+                    "publicKey": {
+                        "content": self.base64_encode(public_key_path)
+                    }
+                },
+                "data": {
+                    "content": base64_encoded_extra_data,
+                    "hash": {
+                        "algorithm": "sha256",
+                        "value": hash
+                    }
+                },
+                "extraData": base64_encoded_extra_data
+            }
+        }
+        return rekor_entry;
+
+    def get_gpg_key(self, sig_file, extra_data_file):
+        """Runs the step implemented by this StepImplementer.
+
+        Returns
+        -------
+        StepResult
+            Object containing the dictionary results of this step.
+        """
+        gpg_stdout_result = StringIO()
+        gpg_stdout_callback = create_sh_redirect_to_multiple_streams_fn_callback([
+            sys.stdout,
+            gpg_stdout_result
+        ])
+        gpg_user = self.get_value('gpg-user')
+        sh.gpg(
+            '--armor',
+            '-u',
+            gpg_user,
+            '--output',
+            sig_file,
+            '--detach-sign',
+            extra_data_file,
+            _out=gpg_stdout_callback,
+            # _in=extra_data,
+            _err_to_out=True,
+            _tee='out'
+        )
+        return gpg_stdout_result
+
+    def upload_to_rekor(self, rekor_server, extra_data_file):
+        """Runs the step implemented by this StepImplementer.
+
+        Returns
+        -------
+        StepResult
+            Object containing the dictionary results of this step.
+        """
+        gpg_key = self.get_value('gpg-key')
+        rekor_entry = self.create_rekor_entry(gpg_key, extra_data_file)
+        print("Rekor Entry: " + str(rekor_entry))
+        # print("Rekor Entry Type: "+ str(type(rekor_entry)))
```

Comment:
get rid of commented out line

Reply:
https://github.com/ploigos/ploigos-step-runner/pull/146/commits/5299e2225db54860ecea7b0e84b5964dc0a428ed